mozilla/foundation.mozilla.org
# Anchor should not wrap the entire 'group listing' card block

(Please ignore the false positive warnings Percy flags.)

# Description

Make sure card rendered from the streamfield block "group listing block"  isn't wrapped in a `<a>`. We want only the image and the title to be hyperlinked and not the entire card.

Related PRs/issues: #9834

---

### To test

- go to any CMS-managed page
- add a streamfield block "group listing block" <img width="258" alt="image" src="https://user-images.githubusercontent.com/2896608/207538411-fc8e8ea0-ded6-4164-9348-2f104facdd78.png">
- fill in all fields
- publish the page and make sure elements are linked properly
